src/www/ui/api/documentation/openapi.yaml

```diff
@@ -681,13 +681,22 @@ paths:
       tags:
         - User
       description: Get the information about logged in user
+
       responses:
         '200':
           description: OK
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/User'
+                allOf:
+                  - type: object
+                    properties:
+                      defaultGroup:
```

Comment:
Value returned from code is `default_group`. Please update the doc accordingly.

Reply:
Updated it